tests/config/test_configtypes.py

```diff
@@ -1378,8 +1378,7 @@ def test_validate_exists_not_abs(self, os_path):
         os_path.expanduser.side_effect = lambda x: x
         os_path.isfile.return_value = True
         os_path.isabs.return_value = False
-        with pytest.raises(configexc.ValidationError):
-            self.t.validate('foobar')
+        self.t.validate('foobar')
```

Comment:
The test for assert_called_once_with() is indeed justified, standarddir.config() actually returns None during that test. I'll try to fix this and the remaining issues tomorrow, I'm out of time for today.